src/main/java/service/converter/DataConverterImpl.java

```diff
@@ -0,0 +1,39 @@
+package service.converter;
+
+import java.util.ArrayList;
+import java.util.List;
+import model.FruitTransaction;
+
+public class DataConverterImpl implements DataConverter {
+    private static final String SEPARATOR = ",";
+    private static final int OPERATION_INDEX = 0;
+    private static final int HEADER_INDEX = 0;
+    private static final int FRUIT_INDEX = 1;
+    private static final int QUANTITY_INDEX = 2;
+
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> inputReport) {
+        List<FruitTransaction> fruitTransactionList = new ArrayList<>();
+        inputReport.remove(HEADER_INDEX);
```

Comment:
Directly modifying the input list by removing an element can lead to unexpected behavior, especially if the list is used elsewhere after this method call. It's safer to skip the header when iterating over the list rather than removing it.